Solutions/Trend Micro Vision One/Data Connectors/AzureFunctionTrendMicroXDR/queue_trigger_wb/__init__.py

```diff
@@ -118,15 +118,17 @@ def customize_json(clp_id, workbench_detail, workbench_record):
     return xdr_log
 
 
-def build_queue_message(clp_id, workbench_id, task_id, task_name, target_guid, target_info):
+def build_queue_message(
+    clp_id, workbench_id, task_id, task_name, target_guid, target_info
+):
     return json.dumps(
         {
             'clp_id': clp_id,
             'workbench_id': workbench_id,
             'task_id': task_id,
             'task_name': task_name,
             'target_guid': target_guid,
-            'target_info': target_info
+            'target_info': target_info,
```

Comment:
Hi @V1ManagedServices , are these comma added needed?

Reply:
Hi @v-sabiraj, These commas are auto-formatted by the Black code style with pre-commit.